mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/KycExchangeRequestDto.java

```diff
@@ -20,4 +20,5 @@ public class KycExchangeRequestDto {
     private String individualId;
     private List<String> acceptedClaims;
     private List<String> claimLocales;
+    private String respType;
```

Comment:
We do not have the field "respType" in the mock-identity-system. Are you planning to introduce v3 version of kyc-exchange? 
Refs -> https://github.com/mosip/esignet-mock-services/blob/develop/mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/KycExchangeRequestDto.java